src/components/@shared/FormInput/InputElement.module.css

```diff
@@ -100,6 +100,10 @@
   padding-left: 0.5rem;
 }
 
+.radioLabelPricing {
+  margin-left: 10px;
```

Comment:
we try to limit magic numbers and absolute units. Ideally, calculate this with `var(--spacer)`, or use `rem` unit

Reply:
ok, got it. This has been remove.